charts/fluentd/values.yaml

```diff
@@ -46,6 +46,24 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
+# Congigure the livessProbe
+# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+livenessProbeConfig: {}
```

Comment:
Just use "livenessProve" same for the other probes. Dont forget to fix template
Any other members that are duplicated are overriten as this comes after.

Reply:
don't understand what you want me to do

Reply:
livenessProbeConfig -> livenessProbe

I dont object how it is, but as you have to replace the helm version anyway....

Reply:
+1 remove the trailing `Config`